### DISCUSSION TITLE
Adonis V5: How to use after in migrations?

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#1518] [Help] [answered]
The [`after()`](http://knexjs.org/#Schema-after) command does not seem to work anymore in Adonis v5.

<img width="609" alt="Screenshot 2020-08-26 at 15 40 17" src="https://user-images.githubusercontent.com/1658644/91310737-892f6100-e7b2-11ea-99a6-19f4d60ecfed.png">

Is there a specific way to use it?

**Comment by thetutlage** [comment, marked as answer]
Issue with knex typings. Opened issue with them https://github.com/knex/knex/issues/4007

**Reply by LeCoupa** [reply]
Thank you 👍
